src/index.ts

```diff
@@ -372,7 +381,16 @@ const oauth2 = <Profiles extends string>({
 
           async tokenHeaders(profile: Profiles) {
             const token = await storage.get(ctx.request, profile);
-            return { Authorization: `Bearer ${token?.access_token}` };
+            return { 
+              Authorization: `Bearer ${token?.access_token}`, 
+              // ! required for twitch
+              'Client-Id': env('TWITCH_OAUTH_CLIENT_ID') 
```

Comment:
not compatible for everyone

Reply:
`env` throws an `Error` if it doesn't exist

Reply:
Fixed.